lib/src/main/java/uk/gov/di/ipv/cri/drivingpermit/library/testdata/PersonIdentityTestDataGenerator.java

```diff
@@ -11,7 +9,9 @@
 
 public class PersonIdentityTestDataGenerator {
 
-    private static final Logger LOGGER = LogManager.getLogger();
+    private PersonIdentityTestDataGenerator() {
```

Comment:
I think if this is private final we dont need the exception
https://stackoverflow.com/questions/14398747/how-to-disable-warning-on-sonar-hide-utility-class-constructor